src/drivers/distance_sensor/sf1xx/sf1xx.cpp

```diff
@@ -857,13 +858,32 @@ info()
 static void
 sf1xx_usage()
 {
-	PX4_INFO("usage: sf1xx command [options]");
-	PX4_INFO("options:");
-	PX4_INFO("\t-b --bus i2cbus (%d)", SF1XX_BUS_DEFAULT);
-	PX4_INFO("\t-a --all");
-	PX4_INFO("\t-R --rotation (%d)", distance_sensor_s::ROTATION_DOWNWARD_FACING);
-	PX4_INFO("command:");
-	PX4_INFO("\tstart|stop|test|reset|info");
+	PRINT_MODULE_DESCRIPTION(
+		R"DESCR_STR(
+### Description
+
+I2C bus driver for Lightware SFxx series LIDAR rangefinders: SF10/a, SF10/b, SF10/c, SF11/c, SF/LW20.
+
+Setup/usage information: https://docs.px4.io/en/sensor/sfxx_lidar.html
+
+### Examples
+
+Attempt to start driver on any bus (start on bus where first sensor found).
+$ sf1xx start -a
+Stop driver
+$ sf1xx stop
+)DESCR_STR");
+
+	PRINT_MODULE_USAGE_NAME("sf1xx", "driver");
+	PRINT_MODULE_USAGE_COMMAND_DESCR("start","Start driver");
+	PRINT_MODULE_USAGE_PARAM_FLAG('a', "Attempt to start driver on all I2C buses ([--all])", true);
+	PRINT_MODULE_USAGE_PARAM_INT('b', 1, 1, 2000, "Start driver on specific I2C bus ([--bus])", true);
+	PRINT_MODULE_USAGE_PARAM_INT('R', 25, 1, 25, "Sensor rotation - downward facing by default ([--rotation])", true);
```

Comment:
Previously this had lower case 'r' but I got an error so I changed it:
```
    Warning: documentation inconsistency in drivers/distance_sensor/sf1xx:
 Documented options       : arb
 Options found in getopt(): aRb
Error: validation failed
```
BUT in either case, when I build and run the option in SITL I get the same error:
```
    pxh> sf1xx start -R
WARN  [sf1xx] Unknown option!
```

Reply:
`sf1xx start -R` you are missing the rotation: `sf1xx start -R 1`